downstream/titles/eda/eda-getting-started-guide/docinfo.xml

```diff
@@ -0,0 +1,10 @@
+<title>Getting Started with Event-Driven Ansible</title>
```

Comment:
This title must match the title in the master.adoc exactly for Pantheon to process it without errors.
Suggest you add Guide here or remove it from the master.adoc file; your choice. ;o)

Reply:
done